tools/capture-test.py

```diff
@@ -352,7 +365,7 @@ def echo_test():
     # Just slurps in the wav file and chops off the header, assuming
     # the user got the format and sampling rate correct.
     WAV_HDR_LEN = 44
-    buf = open(opts.noise, "rb").read()[WAV_HDR_LEN:]
+    buf = open(opts.noise, "rb").read()[WAV_HDR_LEN:] # pylint: disable=consider-using-with
```

Comment:
I don't care in this particular case but I'd really like to get nearer to the bottom of this `with` discussion some day.